.github/workflows/tag.yml

```diff
@@ -0,0 +1,110 @@
+# NB: release notes inspired by https://blogs.sap.com/2018/06/22/generating-release-notes-from-git-commit-messages-using-basic-shell-commands-gitgrep/
```

Comment:
Git commit message contents don't map to a good changelog... https://twitter.com/yetanothertw/status/1197634865988067329

Using Towncrier and forcing PR authors to include human-readable change fragments is a more popular practice that many Python projects use. I even wrote a GitHub App that checks the presence of such fragments in PRs...

Reply:
It's a good point -- and I don't want the actual GitHub release to contain the commits -- just the tag annotated with the list. That way, when I draft a release, I can refer to the changes incorporated in the tag (to remind myself) and write a more appropriate release / Towncrier.